contracts/index.js

```diff
@@ -45,43 +45,37 @@ exports.LIB_ARTIFACT_UTILS_ADDRESS =
 /**
  * The name of the network where these contracts are deployed.
  */
-exports.NETWORK = 'gnosis_optimism';
+exports.NETWORK = 'hardhat';
 /**
  * The id of the network where these contracts are deployed.
  */
-exports.NETWORK_ID = 300;
+exports.NETWORK_ID = 31337;
 /**
  * The block in which the DarkForest contract was initialized.
  */
-exports.START_BLOCK = 48135;
+exports.START_BLOCK = 0;
 /**
  * The address for the DarkForest contract.
  */
-exports.CONTRACT_ADDRESS = '0x9EED3460Ce72254f9F0ad5164C12F39E5E10A40C';
+exports.CONTRACT_ADDRESS = '0x0184E55E6be5c73081E2BdA05d662014E77eA29F';
 /**
  * The address for the initalizer contract. Useful for lobbies.
  */
-exports.INIT_ADDRESS = '0x6b9EA68DbF64aFe8eeFED0169A8B4c5fa8c666cc';
+exports.INIT_ADDRESS = '0x6610d3Db863c2e7c161ECA424e750254409190F2';
 /**
  * The address for the Verifier library. Useful for lobbies.
  */
-exports.VERIFIER_ADDRESS = '0xC6cDe623Dd2a027df6F7c13f7dD240E3A89CBA68';
+exports.VERIFIER_ADDRESS = '0x96E8073F17aD38E2979bbfd0d083aDD08bc2ce2c';
 /**
  * The address for the LibGameUtils library. Useful for lobbies.
  */
-exports.LIB_GAME_UTILS_ADDRESS = '0xFcEF980d2433E335D66a838B86cd213Aa710378D';
+exports.LIB_GAME_UTILS_ADDRESS = '0xEb7081de74B805c45A0D93eec5cF143fbd176b40';
 /**
  * The address for the LibPlanet library. Useful for lobbies.
  */
-exports.LIB_PLANET_ADDRESS = '0x58eD9B2E7f1fEC6889D137Bc7Ced7ecD87aD06Ee';
+exports.LIB_PLANET_ADDRESS = '0xE16337E67a40988CBC71DD9CF07863437eB39c31';
 /**
  * The address for the LibArtifacts library. Useful for lobbies.
  */
-exports.LIB_ARTIFACT_UTILS_ADDRESS = '0x816a0EC0a526968a8d79e7D0e7113d5afAc25eD0';
-'use strict';
```

Comment:
idk why this `use strict` is here, probably a weird relic of the code I'm using to append `FAUCET_ADDRESS` to file